content/kubermatic/master/architecture/support_policy/_index.en.md

```diff
@@ -2,8 +2,17 @@
 +++
 title = "Support Policy"
 date = 2021-04-20T12:11:35+02:00
-weight = 3
+weight = 4
 
 +++
 
-This document describes the supported software and its components, supported operating systems, cloud providers and third-party software. Additionally, it covers the supported infrastructures via their respective API. 
+## Supported KKP Versions
+
+KKP versions are expressed as x.y.z, where x is the major version, y is the
+minor version, and z is the patch version.
+
+KKP follows the Kubernetes release model and cycle, though for practical reasons
+releases are a bit delayed to ensure compatibility with Kubernetes. In general,
+the latest three minor versions of KKP are supported, i.e. 2.19, 2.18 and 2.17.
```

Comment:
Nitpick, can we update this to `i.e. 2.21, 2.20 and 2.19`? This document will be for KKP 2.21, so it seems sensible to list the versions we will actually support at launch.

Reply:
```suggestion
the latest three minor versions of KKP are supported, i.e. 2.21, 2.20 and 2.19.
```